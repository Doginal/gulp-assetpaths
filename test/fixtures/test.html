/* ----- images ---- */

<img src="/images/test.png" />
<img src="http://www.oldDomain.com/images/test.png" />
<img src="images/test.jpg" />
<img src="/images/test.png" /><img src="/images/test.png" />
<img src="http://www.oldDomain.com/images/test.png" /><img src="http://www.oldDomain.com/images/test.png" />
<img src="images/test.jpg" /><img src="images/test.jpg" />
<img src = '/images/test.png' />
<img src = 'http://www.oldDomain.com/images/test.png' />
<img src = 'images/test.jpg' />
<img src = '/images/test.png' /><img src='/images/test.png' />
<img src = 'http://www.oldDomain.com/images/test.png' /><img src='http://www.oldDomain.com/images/test.png' />
<img src = 'images/test.jpg' /><img src='images/test.jpg' />
<div style="background:url(images/test.jpg);" ></div>
<div style="background:url(/images/test.jpg);" ></div>
<div style="background:url('/images/test.jpg');" ></div>
<div style='background: white url("/images/test.jpg");' ></div>
<div style="background-image: url(images/test.jpg);" ></div>
<div style="background-image: url(/images/test.jpg);" ></div>
<div style="background-image: url('/images/test.jpg');" ></div>
<div style='background-image: url("/images/test.jpg");' ></div>
.test {
	background : url(images/test.jpg);
	background : url(/images/test.jpg);
	background : url(https://www.oldDomain.com/images/test.jpg);
}
<img src="http://www.someOtherDomain.com" />

/* ---- css ---- */

<link rel="stylesheet" href="/css/test.css" />
<link rel="stylesheet" href="css/test.css" />
<link rel="stylesheet" href="/css/test.css" /><link rel="stylesheet" href="/css/test.css" />
<link rel="stylesheet" href="css/test.css" /><link rel="stylesheet" href="css/test.css" />
<link rel="stylesheet" href="http://www.someOtherDomain.com" />

/* ---- js ---- */

<script src="/js/test.js"></script>
<script src="js/test.js"></script>
<script src="/js/test.js"></script><script src="/js/test.js"></script>
<script src="https://www.newDomain.com/js/test.js"></script>