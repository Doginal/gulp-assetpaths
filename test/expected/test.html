/* ----- images ---- */

<img src="https://www.newDomain.com/images/test.png" />
<img src="https://www.newDomain.com/images/test.png" />
<img src="https://www.newDomain.com/fixtures/images/test.jpg" />
<img src="https://www.newDomain.com/images/test.png" /><img src="https://www.newDomain.com/images/test.png" />
<img src="https://www.newDomain.com/images/test.png" /><img src="https://www.newDomain.com/images/test.png" />
<img src="https://www.newDomain.com/fixtures/images/test.jpg" /><img src="https://www.newDomain.com/fixtures/images/test.jpg" />
<img src = 'https://www.newDomain.com/images/test.png' />
<img src = 'https://www.newDomain.com/images/test.png' />
<img src = 'https://www.newDomain.com/fixtures/images/test.jpg' />
<img src = 'https://www.newDomain.com/images/test.png' /><img src='https://www.newDomain.com/images/test.png' />
<img src = 'https://www.newDomain.com/images/test.png' /><img src='https://www.newDomain.com/images/test.png' />
<img src = 'https://www.newDomain.com/fixtures/images/test.jpg' /><img src='https://www.newDomain.com/fixtures/images/test.jpg' />
<div style="background:url(https://www.newDomain.com/fixtures/images/test.jpg);" ></div>
<div style="background:url(https://www.newDomain.com/images/test.jpg);" ></div>
<div style="background:url('https://www.newDomain.com/images/test.jpg');" ></div>
<div style='background: white url("https://www.newDomain.com/images/test.jpg");' ></div>
<div style="background-image: url(https://www.newDomain.com/fixtures/images/test.jpg);" ></div>
<div style="background-image: url(https://www.newDomain.com/images/test.jpg);" ></div>
<div style="background-image: url('https://www.newDomain.com/images/test.jpg');" ></div>
<div style='background-image: url("https://www.newDomain.com/images/test.jpg");' ></div>
.test {
	background : url(https://www.newDomain.com/fixtures/images/test.jpg);
	background : url(https://www.newDomain.com/images/test.jpg);
	background : url(https://www.newDomain.com/images/test.jpg);
}
<img src="http://www.someOtherDomain.com" />

/* ---- css ---- */

<link rel="stylesheet" href="https://www.newDomain.com/css/test.css" />
<link rel="stylesheet" href="https://www.newDomain.com/fixtures/css/test.css" />
<link rel="stylesheet" href="https://www.newDomain.com/css/test.css" /><link rel="stylesheet" href="https://www.newDomain.com/css/test.css" />
<link rel="stylesheet" href="https://www.newDomain.com/fixtures/css/test.css" /><link rel="stylesheet" href="https://www.newDomain.com/fixtures/css/test.css" />
<link rel="stylesheet" href="http://www.someOtherDomain.com" />

/* ---- js ---- */

<script src="https://www.newDomain.com/js/test.js"></script>
<script src="https://www.newDomain.com/fixtures/js/test.js"></script>
<script src="https://www.newDomain.com/js/test.js"></script><script src="https://www.newDomain.com/js/test.js"></script>
<script src="https://www.newDomain.com/js/test.js"></script>